lib/sanbase/clickhouse/metric/metric_files/social_metrics.json

```
@@ -90,6 +90,63 @@
     "data_type": "timeseries",
     "docs_links": ["https://academy.santiment.net/metrics/sentiment-metrics/weighted-sentiment-metrics"]
   },
+  {
+    "human_readable_name": "Sentiment Volume Consumed Total V2",
+    "name": "sentiment_volume_consumed_total_v2",
```

```suggestion
    "human_readable_name": "Sentiment Weighted Total V2",
    "name": "sentiment_weighted_total_v2",
```
Expose them as `sentiment weighted`